src/containers/SingleProjectPageContainer/index.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import React from 'react'
+import { Query } from 'react-apollo'
+import ErrorPage from 'next/error'
+import View from '@components/SingleProjectPage'
+import Wrapper from '@components/Wrapper'
+import Loader from '@components/Loader'
+import query from './query.graphql'
+
+type Props = {
+  projectSlug: string
+}
+
+const SingleProjectPageContainer = (props: Props) => (
+  <Wrapper flex contentCentered>
+    <Query query={query} variables={{ projectSlug: props.projectSlug }}>
+      {({ error, loading, data }) => {
+        if (error) {
+          return <ErrorPage statusCode={404} />
```

Comment:
Downside is we don’t render an actual 404… there’s still the sidebar 🤔

Reply:
Little bit weird but... why not 🤷‍♂️

Reply:
I have an idea to improve that in another PR :)